src/components/NavigationAccountExpandCollapse.vue

```diff
@@ -53,6 +54,16 @@ export default {
 	methods: {
 		toggleCollapse() {
 			this.$store.commit('toggleAccountCollapsed', this.account.id)
+			this.$store
+				.dispatch('updateAccountSetting', {
+					accountId: this.account.id,
+					key: 'collapsed',
+					value: this.account.collapsed
```

Comment:
tip: we use ESLint for the js formatting. You can `npm run lint:fix` to have everything in the desired format automatically :)

Reply:
please use async and await (and a try-catch-finally) instead of explicit promises. You'll still find traces of promises in our code base but we generally try to use the modern coding conventions